src/test-run/commands/options.js

```diff
@@ -6,35 +6,35 @@
 import Assignable from '../../utils/assignable';
 
 import {
-    ActionNumberOptionError,
-    ActionPositiveNumberOptionError,
+    ActionIntegerNumberOptionError,
+    ActionPositiveIntegerNumberOptionError,
     ActionBooleanOptionError
 } from '../../errors/test-run';
 
 // Validators
-function number (option, val, positive) {
+function integerNumber (option, val, positive) {
     var valType   = typeof val;
-    var ErrorCtor = positive ? ActionPositiveNumberOptionError : ActionNumberOptionError;
+    var ErrorCtor = positive ? ActionPositiveIntegerNumberOptionError : ActionIntegerNumberOptionError;
 
     if (valType !== 'number')
         throw new ErrorCtor(option, valType);
 
-    if (isNaN(val))
-        throw new ErrorCtor(option, valType);
+    if (isNaN(val) || !Number.isInteger(val))
```

Comment:
babel-transform-runtime polyfills that, below is a part of generated code of `options.js`:
```js
//...
var _isInteger = require('babel-runtime/core-js/number/is-integer');

var _isInteger2 = _interopRequireDefault(_isInteger);
//...

function integerNumber(option, val, positive) {
    var valType = typeof val === 'undefined' ? 'undefined' : (0, _typeof3.default)(val);
    var ErrorCtor = positive ? _testRun.ActionPositiveIntegerNumberOptionError : _testRun.ActionIntegerNumberOptionError;

    if (valType !== 'number') throw new ErrorCtor(option, valType);

    if (isNaN(val) || !(0, _isInteger2.default)(val)) throw new ErrorCtor(option, val);
}

```